umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -72,5 +73,11 @@
         update-format (or update-format (:format concept))
         umm (spec-core/parse-metadata
              context concept-type format metadata {:sanitize? (= :umm-json (mt/format-key update-format))})
-        umm (apply-update update-type umm update-field update-value find-value)]
+        metadata-dates (:MetadataDates umm)
```

Comment:
He asked Kathy about it and she asked to talk to me, so I am waiting to discuss with her. I would like to get an answer from Kathy, who is the creator of the ticket.